src/app/globals.css

```diff
@@ -90,4 +90,10 @@
     textarea::placeholder {
         padding-bottom: 8px;
     }
+}
+
+/* to remove the reveal password button (eye) which is added by MS Edge and IE */
+input::-ms-reveal,
```

Comment:
does the custom one not override the built in one?

Reply:
nope, I can see both without this css line